src/posthog-surveys-types.ts

```diff
@@ -199,6 +199,7 @@ export interface Survey {
     current_iteration: number | null
     current_iteration_start_date: string | null
     schedule?: SurveySchedule | null
+    enable_partial_responses?: boolean | null
```

Comment:
this feels separate to this change... am i being silly

Reply:
it is, but needed anyway since we already have the property on the BE on main posthog so wanted to get it in already